cmd/webhook/main.go

```diff
@@ -100,7 +100,7 @@ func main() {
 			v1alpha1.SchemeGroupVersion.WithKind("TriggerTemplate"): &v1alpha1.TriggerTemplate{},
 		},
 		Logger:                logger,
-		DisallowUnknownFields: true,
+		DisallowUnknownFields: false,
```

Comment:
what is the reason for changing this to false? Not saying it's wrong just trying to understand!

Reply:
The structure of the templated resources is arbitrary, therefore we cannot know what the structure of the data will be. [This](https://github.com/tektoncd/triggers/pull/61/files#diff-590359aadf35a5cd62893ca03297feeeR16) field (or any other) does not have an explicit mapping in the [`TriggerResourceTemplate`](https://github.com/tektoncd/triggers/commit/209ae98bbcf337d3c97eed5b7a124b1ff6052603), which would result in the admission webhook preventing any `TriggerTemplate` from being created.

Reply:
Oddly, the `DisallowUnknownFields` check seems to look for explicit mappings rather than simply checking if the json unmarshals correctly.

Reply:
Hm, I was able to create the `TriggerTemplate` from the examples with `DisallowUnknownFields: true`. But this was from the `master` branch, so maybe something else is different on this branch?

Reply:
I have no idea why it failed for me locally 😒 Happy to put it back